integration-tests/src/test/java/tachyon/master/JournalIntegrationTest.java

```diff
@@ -158,13 +157,26 @@ public final void before() throws Exception {
   }
 
   /**
+   * Assume {@link #before} has been called, then we need to first stop the cluster and start the
+   * cluster again after reconfiguring {@link Constants#MASTER_JOURNAL_MAX_LOG_SIZE_BYTES}.
+   *
+   * @param bytes max log size in bytes
+   */
+  private void configureMaxLogSizeAndRestart(int bytes) throws Exception {
```

Comment:
Can't you achieve the same thing by passing in a TachyonConf into start() of LocalTachyonCluster?